zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1203,6 +1207,30 @@ private String resolveDataVizDataJsonURI(Path contentPath) {
         return contentPath.getParent().resolve(DATA_JSON).toString();
     }
 
+    /**
+     * Get the path of the data viz zip file to delete.
+     * Deleteing a data viz zip is a different process to deleting standard CMS content. Deleting standard CMS
+     * content means deleteing a file so the path to delete will always be a path to a data.json file. However, when
+     * deleteing a data viz zip we have to delete the page (data.json file) and the directory containig the data viz
+     * zip content. We therefore need a path to the parent directory of the data.json file.
+     * <p>
+     * This method will return a path to the parent dir if the uri is a data.json file.
+     * Example:
+     * If uri is "a/b/c/data.json" then it will return "a/b/c".
+     *
+     * If uri is "a/b/c" then it will return "a/b/c".
+     *
+     * @param uri the file path to check.
+     */
+    private String resolveDataVizZipURI(Path uri) {
```

Comment:
Following on from other naming comment, I just wonder if this method is really about a zip dir? In the context it is being used it is, but in isolation this is a method that finds a data json's parent dir unless that dir is root.